wordpress-mobile/WordPress-iOS
# Reader Post: Add receivesCommentNotifications property

Depends on: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/452 👈 Please review this first 🙂 

Adds `receivesCommentNotifications` to `ReaderPost`. This will be used by the upcoming PRs to enable post subscribers to receive push notifications. The default value of the `receivesCommentNotifications` is set to `false` because the push notification feature is opt-out by default, so this checks out.

## To Test

- Go to the Reader section, and verify that all the posts are loaded correctly.
- Tap one of the posts, verify that the post is displayed correctly.

## Regression Notes
1. Potential unintended areas of impact
Potential crash when parsing the new property?

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Ensure that all the screens depending on `ReaderPost` objects work fine.

3. What automated tests I added (or what prevented me from doing so)
n/a.

PR submission checklist:
